docs/modeling-your-data/modeling-your-data-with-dbt/index.md

```diff
@@ -27,39 +27,43 @@ _The Snowplow Media Player package is designed to be used with the Snowplow Web
 
 Each package comes with a set of standard models to take your [Snowplow tracker data](/docs/collecting-data/collecting-from-own-applications/index.md) and produce tables aggregated to levels such as Users, Sessions, and Page Views. You can also add your own models on top, see the page on [custom modules](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-custom-models/index.md) for more information on how to do this.
```

Comment:
Technically `page views` is only for web, so maybe just `views` or `page/screen views`, but then we're excluding media player?

Reply:
I think we just had this as an overview across packages, using page views generically for views as web was the majority package, but with the specific models listed on each specific model page?